plugins/woocommerce/includes/class-wc-deprecated-action-hooks.php

```diff
@@ -175,14 +175,6 @@ protected function trigger_hook( $old_hook, $new_callback_args ) {
 					do_action( $old_hook, $order_id, $item_id, $item, $item->get_product() );
 				}
 				break;
-			case 'woocommerce_order_update_coupon':
```

Comment:
What is this change for?  Is it related to PHP8 updates?

Reply:
It was suggested by you on previous review https://github.com/woocommerce/woocommerce/pull/41253#discussion_r1402041319